server/form-pages/apply/access-and-healthcare/covid.test.ts

```diff
@@ -41,7 +41,7 @@ describe('Covid', () => {
     const page = new Covid({}, application, '')
     expect(page.errors()).toEqual({
       fullyVaccinated: 'You must confirm if John Wayne has been fully vaccinated',
-      highRisk: 'You must confirm if John Wayne is at high risk from COVID-19',
+      highRisk: 'You must confirm if John Wayne is at at a higher risk from COVID-19',
```

Comment:
I think this should be 'You must confirm if John Wayne is at a higher risk from COVID-19'